rq/worker.py

```diff
@@ -591,6 +594,15 @@ def dequeue_job_and_maintain_ttl(self, timeout):
                 break
             except DequeueTimeout:
                 pass
+            except redis.exceptions.ConnectionError as conn_err:
+                self.log.error('Could not connect to Redis instance: %s Retrying in %d seconds...', 
+                                conn_err, connection_wait_time)
+                time.sleep(connection_wait_time)
+                connection_wait_time *= self.exponential_backoff_factor
+                if connection_wait_time > self.max_connection_wait_time:
+                    connection_wait_time = self.max_connection_wait_time
```

Comment:
Nitpick: `connection_wait_time = min(connection_wait_time, self.max_connection_wait_time)`

Reply:
resolved it. thanks for the suggestion.